src/server/utils/sendSegmentEvent.js

```diff
@@ -14,7 +15,54 @@ const getTraits = (userIds) => {
       email: r.row('email').default(''),
       id: r.row('id').default(''),
       name: r.row('preferredName').default(''),
-      parabolId: r.row('id').default('') // passed as a distinct trait name for HubSpot
+      parabolId: r.row('id').default(''), // passed as a distinct trait name for HubSpot
+      // # of orgs the user is on where teams is >= 3
+      salesOpOrgCount: r
+        .table('OrganizationUser')
+        .getAll(r.row('id'), {index: 'userId'})
+        .filter({removedAt: null})('orgId')
+        .default([])
+        .do((orgIds) => {
+          return r
+            .table('Organization')
+            .getAll(r.args(orgIds), {index: 'id'})
+            .merge((row) => ({
+              teamCount: r
+                .table('Team')
+                .getAll(row('id'), {index: 'orgId'})
+                .count()
+            }))
+            .filter((row) => row('teamCount').ge(3))
```

Comment:
3 should be const?

Reply:
yeah, unless we want to use an environment variable, but I'd argue we shouldn't do that until we need to

Reply:
Curious, what would be the need for an env var?

Reply:
That would be if we wanted to change the teamCount threshold without redeploying the app, just changing the server description

Reply:
I agree with @ackernaut tho – let's make `3` a constant called `FREE_TIER_MAX_TEAMS`, set it to `2` and make the comparison `gt()` instead of (`ge()`).

Reply:
Oh, that makes sense!